### DISCUSSION TITLE
G6 calibration error messages

### DISCUSSION THREAD

**Discussion by ToreBj** [NightscoutFoundation/xDrip#1958] [General]
I performed a calibration and had no symptoms of problems in Xdrip+ or AAPS.

I checked the Xdrip log and found this:

![Screenshot_20220125-154824](https://user-images.githubusercontent.com/23278852/151140560-9a937ac4-3761-495c-8c6c-569599a07d29.jpg)

Is this an error? What consequence could it possibly have?

![Screenshot_20220126-103923](https://user-images.githubusercontent.com/23278852/151140826-d5ccbb41-92e4-45b9-a246-e5542a32e967.jpg)

**Comment by Navid200** [comment]
That's on my list:
https://github.com/NightscoutFoundation/xDrip/discussions/1912

My understanding is these come up only if you calibrate shortly after the warm-up ends.
Anyway, my plan is to fix this.  However, I'm not a programmer.  So, anything on my list can take a very long time to fix.  I am sorry about this.

New developers are always welcome to join.

**Reply by emp-00** [reply]
Confirmed, I am also observing this with every G6 sensor start. Not any functionality issue though (only "event log panic") ;-)

**Reply by Navid200** [reply]
You see "Received backfill packet out of sequence" log with every sensor start?
I hope not.

That shows up after a calibration after a restart only for me.  I don't see why you would get it at any other time.

**Reply by emp-00** [reply]
Your're right - the "packet out of sequence" happens with sensor restarts! Sorry for the confusion.

Actually yesterday I started a NEW sensor and this also threw a similar, but not the same error message - exactly at the time when the first readings after the 2h warmup came in:
![image](https://user-images.githubusercontent.com/60714349/151619933-df85754d-d7bd-4b7f-ac1c-12750f3bfd81.png)

**Reply by emp-00** [reply]
However, both errors are not a problem -- all readings are coming in time and the only "issue" is the messages in the logs. In my opinion no priority to "fix" this because it has no bad consequences at least for me.
